docs/tips.md

```diff
@@ -454,7 +454,9 @@ Then save the following script as `~/.config/helix/yazi-picker.sh`:
 ```sh
 #!/usr/bin/env bash
 
-paths=$(yazi --chooser-file=/dev/stdout | xargs -I {} command printf "%q " {})
+paths="$(yazi --chooser-file=/dev/stdout | while read -r; do
+  command printf '%q ' "$REPLY"
+done)"
```

Comment:
```suggestion
paths=$(yazi --chooser-file=/dev/stdout | while read -r; do printf "%q " "$REPLY"; done)
```

It seems like the outer quotes are not necessary, and I found the `printf` is reworked on my system when `xargs` is dropped, so the `command` can be also dropped now.

Reply:
Works on my end too. Thanks!